docs/registry.md

```diff
@@ -2,3 +2,65 @@
 sidebar_position: 3
 title: "Registry"
 ---
+
+# Registry
+
+The registry contains all the services available in the Prem Ecosystem. At https://github.com/premAI-io/prem-registry/tree/main you can see the latest stable version of the registry. We currently expose a UI as well in order for you to easily visualize all the available models https://registry.premai.io/.
+
+## Add a Service
+
+The following steps are necessary in order to submit a new service in Prem Registry.
+
+### 1. Test
+
+Before submitting the service, make sure that the image has been tested with the corresponding Prem Interface. 
+
+> Development Mode To Be Defined. An Idea could be provide external URL in settings.
+
+### 2. Submit
+
+When you are done with testing and you are confident that your service is ready for Prem, you can fork https://github.com/premAI-io/prem-registry/ and add a pull request against `dev` branch. Here the folders and files necessary for the pull request.
```

Comment:
why not main?

Reply:
main is stable, dev has one step more. Prem team tests on dev. We can change the behaviour, but right now it works in this way.